boards/arm/stm32vl_disco/Kconfig.defconfig

```diff
@@ -0,0 +1,38 @@
+# Board configuration
+
+# Copyright (c) 2020 Jonas Eriksson, Up to Code AB
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_STM32VL_DISCO
+
+config BOARD
+	default "stm32vl_disco"
+
+if UART_CONSOLE
+
+config UART_1
+	default y
+
+endif # UART_CONSOLE
+
```

Comment:
Miss a way to enable UART_2 and UART_3.
Please add them under if SERIAL ...

Reply:
Alright. Another pattern copied from stm32f3_disco that seems to be missing from there. I added an if SERIAL

No worries about spotting the issues late; I obviously did not spot them at all..